docs/config.rst

```diff
@@ -202,6 +202,13 @@ Java Kafka clients.
   * Default: 1000
   * Importance: low
 
+``kafka.rest.resource.extension.class``
+  Fully qualified class name of a  valid Implementation of the interface RestResourceExtension. This can be used to inject user defined resources like filters. Typically used to add custom
```

Comment:
nit: capitalization: Implementation -> implementation

nit: might want fully qualified class name so it is easier to find

capability -> capabilities